docs/0006-flat-layout.md

```diff
@@ -0,0 +1,68 @@
+# OCFL Community Extension 0006: Flat OCFL Storage Layout
+
+  * **Extension Name:** 0006-flat-layout
```

Comment:
Should this be `0006-flat-storage-layout` ?

Reply:
Actually, should it be "Flat Direct OCFL Storage Layout"? A layout that had identifier escaping might still be flat but would get around the character limitations

Reply:
fixed